frigate/config/camera/genai.py

```diff
@@ -42,6 +42,11 @@ class GenAICameraConfig(BaseModel):
         default=False,
         title="Save thumbnails sent to generative AI for debugging purposes.",
     )
+    send_after_frames: Optional[int] = Field(
```

Comment:
I prefer the original config structure that you suggested, in case a user does not want the end update. We should also rename `frames` to `significant_updates` as not every frame is sent to the event updated topic by design

Reply:
Thanks for the review, will update